### ISSUE TITLE
Keyboard Focus Outlines not functional

### ISSUE THREAD

**Issue by evan-lauer** [DigitalCarleton/AccessibilityPlus#10]
Anybody else want to test this out and see if they can get the keyboard focus outlines to work? I made mine bright green and the outline still appeared black.
